### ISSUE TITLE
Tab: a11y issues

### ISSUE THREAD

**Issue by mjcarsjens** [nl-design-system/denhaag#414]
Not found by tests but: the way our component seems to work now has the following issue:
The `TabPanel` has an `id` and a `aria-labelledby`, however the `aria-labelledby` points to an `id` which does not exist. The `Tab` button does not have an `id`, which should be the case.

**Comment by mjcarsjens** [comment]
However, if we are going to rework the Tab component to remove the MaterialUI dependency we can just leave this be for now.

**Comment by YourivHDenHaag** [comment]
Tab component is refactored, this issue is solved.
